src/components/uv-metadata-group/uv-metadata-group.tsx

```diff
@@ -20,12 +20,16 @@ export class UVMetadataGroup {
   }
 
   render() {
-    console.log(this.metadata);
     return (
       <div class="group">
         <div class="header">{this.strings.metadataItemHeader}</div>
-        {/* <div class="items">{this.metadataGroupId}</div> */}
-        <div class="items">{this.metadata.resource.id}</div>
+        <div class="items">{
+          (() => {
+            return this.metadata.items.map((item: manifesto.LabelValuePair) => {
+              return <uv-metadata-item label={item.label} value={item.value} />;
+            });
+          })()
+        }</div>
```

Comment:
I think this can be simplified without the wrapping function and it will still work.
```
<div class="items">{
  this.metadata.items.map((item: manifesto.LabelValuePair) => {
    return <uv-metadata-item label={item.label} value={item.value} />;
  })
}</div>
```